### DISCUSSION TITLE
Update useCookie options

### DISCUSSION THREAD

**Discussion by YuLogun** [nuxt/nuxt#17107] [Questions]
Hi!
I'm using `useCookie` to store auth token.
Is there a way to update its options (I'm primarily interested in `expires` or `maxAge`) in a middleware on every route change?
It seems that it's only possible to update the cookie's value

**Comment by Flanker72** [comment]
Use `setCookie` instead, it can update options also.
I use this approach in server middleware exactly for such purposes, it works perfectly.
